include/cynq/ultrascale/hardware.hpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * See LICENSE for more information about licensing
+ *
+ * Copyright 2023
+ * Author: Luis G. Leon-Vega <luis.leon@ieee.org>
+ *         Diego Arturo Avila Torres <diego.avila@uned.cr>
+ *
+ */
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "cynq/amd/accelerator.hpp"
+#include "cynq/enums.hpp"
+#include "cynq/hardware.hpp"
+#include "cynq/status.hpp"
+#include "cynq/xrt/datamover.hpp"
+
+namespace cynq {
+/**
+ * @brief UltraScale class
+ * Provides an interface to access IP Cores in Xilinx FPGAs.
```

Comment:
Add details about compatibility. it should be compatible with ZCU104, ZCU102, ZCU106 and K26